lib/barbeque/message/job_execution.rb

```diff
@@ -13,7 +13,11 @@ def assign_body(message_body)
         super
         @application = message_body['Application']
         @job  = message_body['Job']
-        @body = message_body['Message']
+        @body = if message_body['Message'].is_a?(String)
+                  JSON.parse(message_body['Message'])
```

Comment:
Q. Are there any cases a non-JSON string comes here? In that case, what will be happened?